frontend/app/views/joiner/tierChooser.scala.html

```diff
@@ -1,19 +1,23 @@
 @(
     pageInfo: model.PageInfo,
     eventOpt: Option[model.RichEvent.RichEvent],
-    accessOpt: Option[model.MembershipAccess]
+    accessOpt: Option[model.MembershipAccess],
+    returnUrl: String
 )(implicit token: play.filters.csrf.CSRF.Token)
 
 @import com.gu.membership.salesforce.Tier
 
 @sectionTitle = @{
+    val defaultTitle = "Choose a membership tier to continue"
     accessOpt.map {
         case i if i.isMembersOnly => "You need to be a Guardian member to view this page"
         case i if i.isPaidMembersOnly => "You need to be a Partner or a Patron to view this page"
         case _ => "Choose a membership tier to continue"
-    }.getOrElse(eventOpt.fold("Choose a membership tier to continue")(_.metadata.chooseTier.sectionTitle))
+    }.getOrElse(eventOpt.fold(defaultTitle)(_.metadata.chooseTier.sectionTitle))
```

Comment:
Oh if you're using it in two places then feel free to use a val. I didn't spot the line above.